miniteltest/expectations.go

```diff
@@ -179,6 +180,16 @@ func (ts *TestServer) Wait(max time.Duration) bool {
 	}
 }
 
+// ExpectDone returns a non-nil error when unfulfulled expectations are present in
+// the test server.
+func (ts *TestServer) ExpectDone() error {
```

Comment:
Done in https://github.com/heroku/minitel-go/pull/12/commits/5cc932d342bb76afa075e009934b3bcc7ec8c612